src/modules/sensors/sensor_params_mag.c

```diff
@@ -53,6 +53,15 @@
  * @value 63 Six side calibration
  * @group Sensors
  */
+PARAM_DEFINE_INT32(SENS_MAG_SIDES, 63);
```

Comment:
This could be bits now.

Reply:
The user would then need to select manually which side he wants and I already see ppl not selecting the right ones and having calibration issues...